deep_quoridor/src/train_alphazero.py

```diff
@@ -292,14 +311,16 @@ def main(args):
         type=str,
         help="Parameters for the Agent Evolution Tournament",
     )
+    parser.add_argument(
+        "--sweep",
+        default=None,
+        type=Path,
+        help="Path to WandB sweep config yaml file",
```

Comment:
This command is only used by the wandb agent when it calls this script during a sweep.